README.md

```diff
@@ -63,6 +63,20 @@ built-in LSP client.
   You can enable this feature by calling `setup_client` in your configuration (see
   below).
 
+- Inlay hints (exposed as `:TSInlayHints`/`:TSDisableInlayHints`/`:TSToggleInlayHints`)
+
+  `tsserver` has added experimental support for inlay hints from Typescript v4.4.2
+
+  Supports the following settings:
+
+  - `auto_inlay_hints` (boolean): Set inlay hints on every new buffer visited
```

Comment:
Could we include the default value and mention that users have to modify `init_options` for this to work? If we can be a little more explicit about the exact steps they have to follow to get this to work, it'll save us trouble in the future.